canonical-websites/jaas.ai
# Update dependencies

## Done

Updated npm dependencies

## QA

- Pull feature branch
- Run `./run clean && ./run`
- Verify no console errors
- Click through site
